doc/devel/documenting_mpl.rst

```diff
@@ -707,7 +707,7 @@ Then in any function accepting `~.Line2D` pass-through ``kwargs``, e.g.,
       """
 
 Note there is a problem for `~matplotlib.artist.Artist` ``__init__`` methods,
-e.g., `matplotlib.patches.Patch.__init__`, which supports ``Patch`` ``kwargs``,
```

Comment:
We can't link `__init__` as this is not an explicit entry in the docs. It's contents is folded into the class docstring.